src/commands/data/export/legacy/tree.ts

```diff
@@ -46,15 +47,19 @@ export default class Export extends SfCommand<DataPlanPart[] | SObjectTreeFileCo
   };
 
   public async run(): Promise<DataPlanPart[] | SObjectTreeFileContents> {
+    this.info(
+      'Try the the new "sf data export beta tree" command.  It support SOQL queries with up to 5 levels of objects!'
```

Comment:
if they run the legacy command, you'll get a suggestion for a beta command, it should just b e"data export tree" right
```bash
 ➜  ../../oss/plugin-data/bin/run.js data export legacy tree --query "SELECT ID FROM Account"
Try the the new "sf data export beta tree" command.  It support SOQL queries with up to 5 levels of objects!
```

Reply:
even then, 
```bash
 ➜  ../../oss/plugin-data/bin/run.js data export beta tree --query "SELECT ID FROM Account"   
 ›   Error: command data:export:beta:tree not found
```

Reply:
I'm just going to remove that.  at this point, you're intentionally choosing this old command to avoid the new one.

Reply:
also, you found that I had the "beta" alias wrong on the new commands.